libr/arch/p/fslsp/plugin.c

```diff
@@ -0,0 +1,102 @@
+/* radare - LGPL3 - Copyright 2023 - eibachd, pancake */
+
+#include <r_arch.h>
+
+static bool fslsp_ancmd(RArchSession *s, RAnalOp *op, RArchDecodeMask mask) {
+	char buf_asm[128];
+	ut8 cmd = (r_read_ble32 (op->bytes, R_ARCH_CONFIG_IS_BIG_ENDIAN (s->config)) >> 16) & 0xff;
+
+	switch (cmd) {
+	case 0x00:
+		sprintf (buf_asm, "Block Copy src 0x%x, from 0x%x, to 0x%x, size 0x%x",
+			r_read_ble32 (op->bytes, R_ARCH_CONFIG_IS_BIG_ENDIAN (s->config)) & 0xffff,
+			r_read_ble32 (op->bytes + 4, R_ARCH_CONFIG_IS_BIG_ENDIAN (s->config)),
+			r_read_ble32 (op->bytes + 8, R_ARCH_CONFIG_IS_BIG_ENDIAN (s->config)),
+			r_read_ble32 (op->bytes + 12, R_ARCH_CONFIG_IS_BIG_ENDIAN (s->config)));
+		op->type = R_ANAL_OP_TYPE_MOV;
+		op->size = 16;
+		break;
+
+	case 0x10:
+		sprintf (buf_asm, "Load RCW with Checksum");
```

Comment:
I really would like to dump the RCW block here. What would be the proper way to do so?

Reply:
how big is this rcw block? imho you can do that using an anal hint later, or associating a type or a format string to the given address where the instruction is found.

Reply:
It is 32x 32bit words. Is there an example how to do the anal hint?